examples/build.gradle

```diff
@@ -29,20 +29,29 @@ configurations {
   unresolvable2
 }
 
+def isNonStable = { String version ->
+  def stableKeyword = ['RELEASE', 'FINAL', 'GA'].any { it -> version.toUpperCase().contains(it) }
+  def regex = /^[0-9,.v-]+$/
+  return !stableKeyword && !(version ==~ regex)
+}
+
 dependencyUpdates {
   checkForGradleUpdate = true
-  resolutionStrategy {
-    componentSelection { rules ->
-      rules.all { ComponentSelection selection ->
-        boolean rejected = ['alpha', 'beta', 'rc', 'cr', 'm', 'preview', 'b', 'ea'].any { qualifier ->
-          selection.candidate.version ==~ /(?i).*[.-]$qualifier[.\d-+]*/
-        }
-        if (rejected) {
-          selection.reject('Release candidate')
-        }
-      }
-    }
+
+  rejectVersionIf { selection ->
+    isNonStable(selection.candidate.version) && !isNonStable(selection.currentVersion)
   }
+
+//  resolutionStrategy {
+//    componentSelection { rules ->
+//      rules.all {
+//        if (isNonStable(candidate.version) && !isNonStable(currentVersion)) {
+//          selection.reject('Release candidate')
```

Comment:
This is same as in readme. If you uncomment this, it would not work, because there is no `selection` variable.